javascript/javascript1/week3/warmuptask.js

```diff
@@ -0,0 +1,70 @@
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+function removeTheName(name) {
+  return name !== nameToRemove;
+}
+const newArray = names.filter(removeTheName);
+console.log(newArray);
+
+//When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+function calcTravelTime(travelInfo) {
+  let time = travelInfo["destinationDistance"] / travelInfo["speed"];
+  const hours = parseInt(time);
```

Comment:
If a number is divided by another number then the result will be a number. You don't need to add the `parseInt()` here.

Reply:
ok, Changed